Mathlib/SetTheory/Cardinal/Ordinal.lean

```diff
@@ -1452,41 +1452,16 @@ section Initial
 
 namespace Ordinal
 
-open Cardinal
-
 /--
-`initial o` (notation: `ω_ o`) gives the first ordinal of cardinality
-`aleph o`. Thus `ω_ 0 = ω` and `ω_ 1` is the first uncountable ordinal.
-You can also write `ω_ 1` as `ω₁`.
+`ω_ o` is a notation for the *initial ordinal* of cardinality
+`aleph o`. Thus, for example `ω_ 0 = ω`.
 -/
-def initial (o : Ordinal.{u}) : Ordinal.{u} := (aleph o).ord
-
-@[inherit_doc]
-scoped notation "ω_" => Ordinal.initial
-@[inherit_doc]
-scoped notation "ω₁" => Ordinal.initial 1
-
-@[simp]
-theorem card_initial (o : Ordinal): card (initial o) = aleph o := card_ord _
+scoped notation "ω_" => λ o ↦ ord <| aleph o
 
 /--
-The first (infinite) initial ordinal is `ω`.
+`ω₁` is the first uncountable ordinal.
 -/
-@[simp]
-lemma initial0 : ω_ 0 = ω := by
-  unfold initial
-  simp
-
-/--
-An initial ordinal is a limit ordinal.
--/
-lemma isLimit_initial (o : Ordinal): (ω_ o).IsLimit := ord_isLimit (aleph0_le_aleph o)
-
-lemma omega_lt_omega1 : ω < ω₁ := ord_aleph0.symm.trans_lt (ord_lt_ord.mpr (aleph0_lt_aleph_one))
```

Comment:
These are some interesting lemmas but I'm also fine with it if you don't want to keep them.

Reply:
It seems that the first 3 are indeed redundant once the definition layer was stripped, but perhaps the last one is worth keeping. I'll put that one back.